delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/model/ResourceRejection.scala

```diff
@@ -198,6 +198,12 @@ object ResourceRejection {
     */
   final case class SchemaIsDeprecated(schemaId: Iri) extends ResourceRejection(s"Schema '$schemaId' is deprecated.")
 
+  /**
+    * Rejection returned when attempting to update the resource with the schema it is already constrained with.
+    */
+  final case class IdenticalSchema()
```

Comment:
can be a case object

Reply:
You might have to put `rejectctedWith[IdenticalSchema.type]`, this can be gotten around by making an empty trait also called IdenticalSchema. I can't say whether it's worth it but it's possible anyway

Reply:
I have removed the test since it was not needed so now it's just a case object :macgyver:

Reply:
Resolving this since it's no longer relevant